api/src/api/v1/auth.ts

```diff
@@ -38,8 +38,6 @@ export default async (server: Hapi.Server) => {
    * Standard strategy
    */
   server.auth.strategy('standard', 'session_cookie', { validate: standardStrategy.validate });
-  const cleanup = standardStrategy.monitorSessionExpiry(knex, cache.session.options.url);
```

Comment:
Why has this been removed?

Reply:
Remember we spoke about removing this because of #302 and the fact that it was potentially the cause of flaky tests on CI?